core/src/bitcoin/bech32/Bech32Parameters.h

```diff
@@ -76,7 +76,7 @@ namespace ledger {
             };
             extern LIBCORE_EXPORT const Bech32Struct getBech32Params(const std::string &networkIdentifier);
             extern LIBCORE_EXPORT const std::vector<Bech32Struct> ALL;
-            static bool insertParameters(soci::session& sql, const Bech32Struct &params);
+            extern LIBCORE_EXPORT bool insertParameters(soci::session& sql, const Bech32Struct &params);
```

Comment:
Why you change this?
Now you expose to the client of the library soci::session&, I think we need to avoid this, as it compexify the interface, and this is not ABI safe

Reply:
To fix link issues, it is now used in another build unit, knowing that `static` method when declared under a namespace have internal linkage.

Reply:
So just remove static, `bool insertParameters(soci::session& sql, const Bech32Struct &params);`